### ISSUE TITLE
aria-hidden-focus rule: one of the test cases have wrong outcome.

### ISSUE THREAD

**Issue by jkodu** [auto-wcag/auto-wcag#346]
For the failed example 3 in rule, believe it should be a pass case, for `aria-hidden` can't be reset once set it to true on an ancestor.

```html
<div aria-hidden="true">
    <div aria-hidden="false">
        <button tabindex="-1">Some button</button>
    </div>
</div>
```

**Comment by jkodu** [comment]
@annethyme - can I get your opinion on this?

**Comment by jkodu** [comment]
PR in place. Closing the issue.
